proxy/http/HttpProxyServerMain.cc

```diff
@@ -321,3 +321,10 @@ start_HttpProxyServerBackDoor(int port, int accept_threads)
   // The backdoor only binds the loopback interface
   netProcessor.main_accept(new HttpSessionAccept(ha_opt), NO_FD, opt);
 }
+
+void
+stop_HttpProxyServer()
+{
+  sslNetProcessor.stop_accept();
+  netProcessor.stop_accept();
+}
```

Comment:
As for the assert, it seems the NetVConnection has to be closed with `close_UnixNetVConnection()` before calling its free. Otherwise, `net_connections_currently_open_stat` won't be decremented.